library/src/scala/util/FromDigits.scala

```diff
@@ -20,11 +23,13 @@ trait FromDigits[T] {
   def fromDigits(digits: String): T
 }
 
+@experimental
 object FromDigits {
 
   /** A subclass of `FromDigits` that also allows to convert whole number literals
    *  with a radix other than 10
    */
+  @experimental
```

Comment:
Is it necesary to mark these definitions as experimental since their enclosing object is already experimental?

Reply:
It is by the inheritance check. We could add an exception, but then we would need the annotation inference again or make the `isExperimental` more computationally expensive.

Reply:
The best solution I found is to infer all `@experimental` on classes nested in an `@experimental` definition and then require the annotation on any class that extends an experimental class. This implies that if we have a class that is both nested in experimental class and extends an experimental class it will infer the annotation. This also implies that we only need to look at the direct parents to do the checks.